test/DebuggerCommon/async.js.dbg.baseline

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "callStack": [
+      {
+        "line": 10,
+        "column": 4,
+        "sourceText": "return await af1(10)",
+        "function": "af2"
+      },
+      {

```

Comment:
Yes, nice to have that... 

---
In reply to: [82048419](https://github.com/Microsoft/ChakraCore/pull/1691#discussion_r82048419) [](ancestors = 82048419)